### PR TITLE
[Docs] mention curl2scrapy in Request.from_curl docs

### PR DESCRIPTION
**Pull request by aditi137**
Cover https://michael-shub.github.io/curl2scrapy/ in the documentation #4206
Create scrapy command to get the equivalent Request from a cURL command #3991

### CODE REVIEW

File: docs/topics/developer-tools.rst [hunk 1]
```diff
@@ -292,6 +292,9 @@ Alternatively, if you want to know the arguments needed to recreate that
 request you can use the :func:`scrapy.utils.curl.curl_to_request_kwargs`
 function to get a dictionary with the equivalent arguments.
 
+Note that to translate a cURL command to a scrapy request,
```
Comment (Gallaecio) [inline docs/topics/developer-tools.rst]:
```suggestion
Note that to translate a cURL command into a Scrapy request,
```

File: docs/topics/developer-tools.rst [hunk 2]
```diff
@@ -292,6 +292,9 @@ Alternatively, if you want to know the arguments needed to recreate that
...
+you may use `curl2scrapy <https://michael-shub.github.io/curl2scrapy/>`_
```
Comment (Gallaecio) [inline docs/topics/developer-tools.rst]:
```suggestion
you may use `curl2scrapy <https://michael-shub.github.io/curl2scrapy/>`_.
```

**Comment by Gallaecio** [comment]
@aditi137 I think something went wrong when merging the latest changes from `master`, the diff should not include the removal of `object` from class definitions.

Could you try merging the latest changes from `master`, or rebasing your changes on top of them? If you are not sure how to, you will find links to useful GitHub documentation related to Git at http://gsoc2020.scrapinghub.com/participate#pre-application-pull-request

**Comment by aditi137** [comment]
Thanks for replying @Gallaecio! Yes I figured out what went wrong. I am creating a new PR here: https://github.com/scrapy/scrapy/pull/4442 and closing this one.

**Pull request closed without merging**